### PR TITLE
Completed javascript-operators-and-expressions exercise - Justin Song

### PR DESCRIPTION
**Pull request by justinsong1608**
Screenshot 2023-01-30 at 12 01 08 PM

### CODE REVIEW

File: javascript-operators-and-expressions/main.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+console.log('value of isAcidic: ', isAcidic);
+console.log('typeof isAcidic: ', typeof isAcidic);
+
+var headCount = 11;
+var isSparta = headCount === 300;
+console.log('value of isSparta: ', isSparta);
+console.log('typeof isSparta: ', typeof isSparta);
+
+var motto = fullName;
+motto = motto + ' is the GOAT';
```
Comment (shawnkost) [inline javascript-operators-and-expressions/main.js]: Use the addition assignment operator here. `+=`.

**Review by shawnkost — CHANGES_REQUESTED** [review]
Hey Justin! One small change below

File: javascript-operators-and-expressions/main.js (hunk 1 above)
Comment (justinsong1608) [reply javascript-operators-and-expressions/main.js]: Updated!

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**